core/src/main/cpp/gdn/projection_mesh/equirectangular_projection_mesh.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CORE_SRC_MAIN_CPP_GDN_PROJECTION_MESH_EQUIRECTANGULAR_PROJECTION_MESH_H_
+#define CORE_SRC_MAIN_CPP_GDN_PROJECTION_MESH_EQUIRECTANGULAR_PROJECTION_MESH_H_
```

Comment:
Can you update these to reflect the pattern used.

Reply:
Done.